### DISCUSSION TITLE
how to set up a message summary?

### DISCUSSION THREAD

**Discussion by egret135** [PHPMailer/PHPMailer#2541] [Q&A] [answered]
I want to set up email summary, but I find that this function is not available, or I do not find this function.

**Comment by Synchro** [comment, marked as answer]
I don't know what you mean. If you want to use the content you used to create a message for something else, you'll need to do that yourself as it has nothing to do with building or sending email.

**Reply by egret135** [reply]
There is a mail summary under the mail header in the general mail list. I want to set this summary when sending, but I don't know how to set it.

I want to send you a screenshot, but I don't know how to attach the picture. Sorry.

**Reply by Synchro** [reply]
You can just paste an image into the text editor on this page. Generally speaking, anything to do with appearance is an app-level concern, not something that PHPMailer is involved with.

**Reply by egret135** [reply]
Thank you for your reply. I have found a solution. Thank you again!
